content/actions/learn-github-actions/migrating-from-circleci-to-github-actions.md

```diff
@@ -107,7 +107,7 @@ GitHub Actions
 </tr>
 </table>
 
-{% data variables.product.prodname_actions %} caching is only applicable to {% data variables.product.prodname_dotcom %}-hosted runners. For more information, see "<a href="/actions/guides/caching-dependencies-to-speed-up-workflows" class="dotcom-only">Caching dependencies to speed up workflows</a>."
+{% data variables.product.prodname_actions %} caching is only applicable for repositories hosted on {% data variables.product.prodname_dotcom %}.com. For more information, see "<a href="/actions/guides/caching-dependencies-to-speed-up-workflows" class="dotcom-only">Caching dependencies to speed up workflows</a>."
```

Comment:
```suggestion
{% data variables.product.prodname_actions %} caching is only applicable for repositories hosted on {% data variables.product.prodname_dotcom_the_website %}. For more information, see "<a href="/actions/guides/caching-dependencies-to-speed-up-workflows" class="dotcom-only">Caching dependencies to speed up workflows</a>."
```
Thanks so much for following up with a PR @Maks3w! I think using this variable from `data/variables/product.yml` should render correctly. Let me know when you commit this suggestion and I'll get this merged ✨

Reply:
Done. I suspect some variable should exists but I think it was on the index were it was composed with the `.com` sufix